gui-wm/wayfire/wayfire-0.7.5.ebuild

```
@@ -0,0 +1,104 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson toolchain-funcs
+
+DESCRIPTION="compiz like 3D wayland compositor"
+HOMEPAGE="https://github.com/WayfireWM/wayfire"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/WayfireWM/${PN}.git"
+else
+	SRC_URI="https://github.com/WayfireWM/${PN}/releases/download/v${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~riscv ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="debug +gles +system-wfconfig +system-wlroots X"
+
+DEPEND="
+	dev-libs/libinput:=
+	dev-libs/wayland
+	gui-libs/gtk-layer-shell
+	media-libs/glm
+	media-libs/mesa:=[gles2,wayland,X?]
+	media-libs/libglvnd[X?]
+	media-libs/libjpeg-turbo:=
+	media-libs/libpng:=
+	media-libs/freetype:=[X?]
+	x11-libs/libdrm
+	x11-libs/gtk+:3=[wayland,X?]
+	x11-libs/cairo[X?,svg(+)]
+	x11-libs/libxkbcommon[X?]
+	x11-libs/pango
+	x11-libs/pixman
+	X? (
+		x11-base/xwayland
+		x11-libs/libxcb
+	)
+	system-wfconfig? (
+		=gui-libs/wf-config-0.7*
+	)
+	!system-wfconfig? ( !gui-libs/wf-config )
+	system-wlroots? (
+		>=gui-libs/wlroots-0.16.0:0/16[X?]
+	)
+	!system-wlroots? ( !gui-libs/wlroots )
+"
+
+RDEPEND="
+	${DEPEND}
+	x11-misc/xkeyboard-config
+"
+
+BDEPEND="
+	dev-libs/wayland-protocols
+	virtual/pkgconfig
+"
+
+src_configure() {
+	sed -e "s:@EPREFIX@:${EPREFIX}:" \
+	    "${FILESDIR}"/wayfire-session > "${T}"/wayfire-session || die
+	sed -e "s:@EPREFIX@:${EPREFIX}:" \
+	    "${FILESDIR}"/wayfire-session.desktop > "${T}"/wayfire-session.desktop || die
+	local emesonargs=(
+		$(meson_feature system-wfconfig use_system_wfconfig)
+		$(meson_feature system-wlroots use_system_wlroots)
+		$(meson_feature X xwayland)
+		$(meson_use gles enable_gles32)
+		$(usex debug --buildtype=debug "")
```

EAPI 8 would allow the use of `usev` with 2 arguments instead of the ugly `usex` with a 3rd empty argument (here and 2 more times below)

```suggestion
		$(usev debug --buildtype=debug)
```

---

All `usex` replaced by `usev` without empty args. I took notes on this aspect for future ebuild work.